### PR TITLE
Improve install doc

### PR DESCRIPTION
**Pull request by matks**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Improve install.txt file content
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | No need for QA

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/16241)

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
Need a second review

**Comment by matks** [comment]
> Need a second review

Even for changes in a TXT file 😅?

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thanks @matks

**Pull request merged**
